crt_portal/utils/markdown_extensions.py

```diff
@@ -107,6 +105,34 @@ def run(self, lines: list[str]) -> list[str]:
         for start, end in should_remove:
             raw_markdown = raw_markdown[:start] + raw_markdown[end:]
 
+        return raw_markdown
+
+    def _preview(self, raw_markdown, optionals) -> str:
+        to_highlight = sorted([
+            (option['start_char'], option['end_char'], option)
+            for group, options in optionals.items()
+            for option in options
+        ], key=lambda startend: -startend[0])
+
+        for start, end, option in to_highlight:
+            # This must be a single line; whitespace throws off the markdown:
+            raw_markdown = (
+                f"{raw_markdown[:start]}\n\n<details><summary>"
+                f"[Optional] {option["group"]}: {option["name"]}</summary>"
```

Comment:
When I tried to start the server I got an error related to this. When I changed group, name, and content to single quotes it worked.
<img width="734" alt="Screenshot 2024-04-16 at 9 07 19 AM" src="https://github.com/usdoj-crt/crt-portal/assets/18104884/c46fcd53-a55b-4bbf-9f2f-9c2b2dac8c99">

Reply:
Oh wow good catch. I am officially confused as to how this ever worked - but I have a screenshot above that shows it somehow did :|